src/Wizard.js

```diff
@@ -733,10 +733,14 @@ export default class Wizard extends Webform {
     if (!form) {
       return;
     }
-    this.wizard = form;
```

Comment:
My only issue is that this will now perform a double clone deep since it passes this into the webform (which also performs a cloneDeep). For large wizards, we will see a performance penalty for this. Can we pass a flag into ```setForm``` called "reference" to tell the setForm method to not perform the cloneDeep? Like this?

```
return super.setForm(wizardForm, {...flags, reference: true})
```